sdk/security/__tests__/validateTokenSecurity.test.js

```diff
@@ -72,6 +72,31 @@ describe('validateToken', () => {
     });
   });
 
+  it('not validates token readSafeJSONString not works', async () => {
```

Comment:
le cambiaría el nombre del test a 'does not validate token when readSafeJSONString throws error' :)

Reply:
listo!